src/styles/_base.scss

```diff
@@ -29,6 +29,12 @@ a {
   width: 100vw;
 }
 
+#map-pattern-fills {
+  // the svg for the map's pattern fills should be hidden in the UI
```

Comment:
Interesting way to deal with this.

Reply:
Maybe I should move that block to the map sass partial